cristin-person/src/test/java/no/unit/nva/cristin/person/handler/userAdministrator.feature

```diff
@@ -39,13 +39,12 @@ Feature: API tests for Cristin Person fetch
     Then status 200
     And response.NationalIdentificationNumber != '#present'
 
-  Scenario: Fetch returns 401 Unauthorized when token is invalid token
+  Scenario: Fetch returns 200 OK and no NationalIdentificationNumber when token is invalid
     Given path '/person/' + samplePersonIdentifier
     * header Authorization = 'Bearer and.just-a.silly-text-for-token'
     When method GET
-    Then status 401
-    And match response.title == 'Unauthorized'
-#    And match responseHeaders['WWW-Authenticate'][0] == 'WWW-Authenticate'
+    Then status 200
+    And response.NationalIdentificationNumber == '#present'
```

Comment:
Does this have a value?

Reply:
I think you need to use `And match response.xxx` and also `NationalIdentificationNumber` is a field in list `identifiers` but it also has gotten a flat field directly in the model